app/config/messaging.js

```diff
@@ -0,0 +1,64 @@
+import environments from '../constants/environments.js'
+
+const messaging = {
+  sharedConfig: {
+    host: {
+      doc: 'Message queue host.',
+      format: String,
+      default: null,
+      env: 'MESSAGE_QUEUE_HOST'
+    },
+    username: {
+      doc: 'Message queue username.',
+      format: String,
+      default: null,
+      nullable: true,
+      env: 'MESSAGE_QUEUE_USER'
+    },
+    password: {
+      doc: 'Message queue password.',
+      format: String,
+      default: null,
+      nullable: true,
+      env: 'MESSAGE_QUEUE_PASSWORD'
+    },
+    useCredentialChain: {
+      doc: 'Use of credential chain for authentication.',
+      format: Boolean,
+      default: process.env.NODE_ENV === environments.PRODUCTION
+    },
+    managedIdentityClientId: {
+      doc: 'Client ID of the managed identity for the service.',
+      format: String,
+      default: null,
+      nullable: true,
+      env: 'AZURE_CLIENT_ID'
+    },
+    appInsights: {
+      doc: 'App Insights client instance.',
+      format: '*',
+      default: process.env.NODE_ENV === environments.PRODUCTION ? await import('applicationinsights') : undefined
+    }
+  },
+  receiverSubscription: {
+    address: {
+      doc: 'Receiver subscription address (i.e. name of subscription).',
+      format: String,
+      default: null,
+      env: 'MESSAGE_QUEUE_SUFFIX'
+    },
+    topic: {
+      doc: 'Receiver topic address (i.e. name of topic corresponding to the subscription).',
+      format: String,
+      default: null,
+      env: 'MESSAGE_QUEUE_SUFFIX'
```

Comment:
I agree that we probably need to rename this but something like `COMMS_TOPIC_NAME` would be better.

Reply:
Going to implement a combination of both you and John's suggestion so that we have a `MESSAGE_QUEUE_SUFFIX` that forms part of the names for two environment variables `COMMS_TOPIC_ADDRESS` and `COMMS_SUBSCRIPTION_ADDRESS` - this should then be consistent for what I've seen in other repos across the programme.